app/models/version_metadata.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+# Models optional metadata that is associated with a druid/version pair for any workflow
+class VersionMetadata < ApplicationRecord
```

Comment:
Can we rename to VersionContext ? "Metadata" is very overused and I believe it will be confusing to talk about.

Reply:
OK.  Renaming all references to `VersionContext` and attribute used from `metadata` to `context`.  Will update draft PR of `dor-workflow-client` as well.